projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -705,6 +705,16 @@ public static function enqueue_block_editor_assets() {
 				'allowedMimeTypes' => wp_get_mime_types(),
 			)
 		);
+
+		/**
+		 * Adds an inline script which updates the block editor settings to
+		 * add the site locale. This feels sligktly better than calling back
+		 * to the API before registering the block. It also seemed better than
```

Comment:
> It also seemed better than creating a global

At this point, would it be worth reconsidering this? I'm asking because this seemed a bit hacky, and this may be the opportunity to reconsider this implementation, since we know now that we need `siteLocale` for more than one block?

Reply:
Sure, I could look into it and see what options we have. 

We were already using the siteLocale for multiple blocks, in this PR  we just moved it from Opentable to a more generic place, next to where we actually enqueue the asset.